niri-ipc/src/lib.rs

```diff
@@ -257,6 +257,15 @@ pub enum SizeChange {
     AdjustProportion(f64),
 }
 
+/// Workspace reference (index or name) to operate on.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
```

Comment:
Guess this can be `Eq` too.
```suggestion
#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
```